src/nomad_catalysis/schema_packages/catalysis.py

```diff
@@ -0,0 +1,378 @@
+)
+from nomad.metainfo.metainfo import Category
+
+# from .catalyst_measurement import (
+#     CatalyticReactionData,
+#     CatalyticReactionData_core,
+#     Rates,
+#     ReactionConditions,
+#     ReactionConditionsSimple,
+#     ReactorSetup,
+#     add_activity,
+# )
+# from .catalyst_measurement import Product as Product_data
+# from .catalyst_measurement import Reactant as Reactant_data
+# from .catalyst_measurement import Reagent as Reagent_data
+
+if TYPE_CHECKING:
+    from nomad.datamodel.datamodel import (
+        EntryArchive,
+    )
+    from structlog.stdlib import (
+        BoundLogger,
+    )
+
+
+configuration = config.get_plugin_entry_point(
+    'nomad_catalysis.schema_packages:catalysis'
+)
+
+m_package = SchemaPackage()
+
+
+class CatalysisElnCategory(EntryDataCategory):
+    m_def = Category(label='Catalysis', categories=[EntryDataCategory])
+
+
+def add_catalyst(archive: 'EntryArchive') -> None:
+    '''
+    Adds metainfo structure for catalysis data to the results section of the supplied
+    archive.
+    '''
+    if not archive.results:
+        archive.results = Results()
+    if not archive.results.properties:
+        archive.results.properties = Properties()
+    if not archive.results.properties.catalytic:
+        archive.results.properties.catalytic = CatalyticProperties()
+    if not archive.results.properties.catalytic.catalyst:
+        archive.results.properties.catalytic.catalyst = Catalyst()
+
+
+def get_nested_attr(obj, attr_path):
+    '''helper function to retrieve nested attributes'''
+    for attr in attr_path.split('.'):
+        obj = getattr(obj, attr, None)
+        if obj is None:
+            return None
+    return obj
+
+
+def set_nested_attr(obj, attr_path, value):
+    '''helper function to set nested attributes'''
+    for attr in attr_path.split('.'):
+        obj = getattr(obj, attr, None)
+        if obj is None:
+            return
+    setattr(obj, attr[-1], value)
+
+
+class Preparation(ArchiveSection):
+    preparation_method = Quantity(
+        type=str,
+        shape=[],
+        description="""
+          Classification of the dominant preparation step
+          in the catalyst synthesis procedure.
+          """,
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'precipitation',
+                    'hydrothermal',
+                    'flame spray pyrolysis',
+                    'impregnation',
+                    'calcination',
+                    'unknown',
+                ]
+            ),
+            links=['https://w3id.org/nfdi4cat/voc4cat_0007016'],
+        ),
+    )
+
+    preparation_entry_reference = Quantity(
+        type=Process,
+        shape=[],
+        description="""
+        A reference to the entry that contains the details of the preparation method.
+        """,
+        a_eln=dict(component='ReferenceEditQuantity'),
+    )
+
+    preparator = Quantity(
```

Comment:
Maybe check name in Voc4cat

Reply:
This is what it is also called in the FHI internal database, so I am keeping it for now.